src/main/resources/Ideas/IdeasSheet.xml

```diff
@@ -356,9 +372,48 @@
   color: $theme.linkColor;
   cursor: pointer;
   text-decoration: underline;
+}
+
+.recent {
+  line-height: 30px;
+}
+
+.recent .xvote-pro, .recent .xvote-con {
+  margin: .1em .3em .5em 0;
+  width: 15px;
+  height: 15px;
+  line-height: 15px;
+  font-size: 15px;
+  padding: 1px;
+  border-radius: 0;
+  font-size: .8em;
+}
+
+.recent .noitems {
+  font-size: inherit;
```

Comment:
Yes, because I think this:
![macro2](https://cloud.githubusercontent.com/assets/8993426/5875268/fa8a982c-a313-11e4-9734-6bc8747473f8.png)
Looks better than this: 
![macro](https://cloud.githubusercontent.com/assets/8993426/5875269/00f872a6-a314-11e4-9f53-755d8b334478.png)